Src/Format.hs

```diff
@@ -1,23 +1,31 @@
 module Format where
+{- Description: various utilities for nicely formatting things,
+  including terms, debugging info and various raw Strings
```

Comment:
They're formats as in format strings for `printf`, hence the `StringPart` too.

Reply:
Yeah, I should give more details in the docs, as it's readable from the code what's going on.